README.md

```diff
@@ -14,6 +14,12 @@ To install a particular version of `wasm-pack`:
 cargo install --version 0.10.2 wasm-pack
 ```
 
+Note: when you run the command in the project directory, you may run into an error `The package requires the Cargo feature called edition2021, but that feature is not stablized in this version of Cargo (1.56.0-nightly (cc17afbb0 2021-08-02)).` 
+
+This is because `rust-toolchain` specifies an overriding cargo version of `nightly-2021-08-12`.
+
+To fix this, run `cargo install --version 0.10.2 wasm-pack` again outside the current directory. Installation was successful usgin cargo `cargo 1.65.0-nightly (4ed54cecc 2022-08-27)`.
```

Comment:
Small typo here, should be *using* I think.

Reply:
Fixed typo and reworded the phrase slightly